### ISSUE TITLE
"NO xml in atomInfo.xml" when running segment validation tests

### ISSUE THREAD

**Issue by nicholas-fr** [Dash-Industry-Forum/DASH-IF-Conformance#612]
When running the tool via a Docker image build with the latest development branch (https://github.com/Dash-Industry-Forum/DASH-IF-Conformance/commit/626fe38b47b1988b07249e8673e0f82d8cb571ce), on the CTA WAVE test vectors, with segment validation enabled, the following unexpected message appears: 
`NO xml in /var/www/html/Utils/sessions/[...]/Period0/Adaptation0/Representation0/atomInfo.xml`

Example: 
```
php Process_cli.php --cmaf --ctawave --segments https://dash.akamaized.net/WAVE/vectors/cfhd_sets/12.5_25_50/t34/2022-10-17/stream.mpd
Going to run segment validator with config file /var/www/html/Utils/sessions/1667985563/Period0/Adaptation0/Representation0/segmentValidatorConfig.txt
NO xml in /var/www/html/Utils/sessions/1667985563/Period0/Adaptation0/Representation0/atomInfo.xml
```

The "NO xml" message might be caused by the lack of a root xml element in atomInfo.xml, unless it is another known issue.

```
root@80d4d1513f39:/var/www/html/Utils/sessions/1667985563/Period0/Adaptation0/Representation0# cat atomInfo.xml
<atomlist>
  <ftyp majorbrand="iso6" version="0x1" compatible_brands='[
     "cfhd"
     "cmf2"
     "iso6"
     "dash"
    ]'>
  </ftyp>
[...]
</atomlist>
```

**Comment by dsilhavy** [comment]
@Phencys analyzed that and found the root cause. It seems to be caused by an invalid XML tag. The fix will be part of the internal refactor phase 2 and milestone 2.1.0.

**Comment by Phencys** [comment]
I have created a pull request that should both clarify the actual meaning of this message if it occurs, and prevent if from occuring at this point in the first place.

**Comment by dsilhavy** [comment]
Fixed in #616, please comment if you still encounter this, closing for now.
